src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const mainImage = document.getElementById('largeImg');
+
+const allLinks = document.getElementsByTagName('a');
+
+for (let i = 0; i < allLinks.length; i++) {
+  allLinks[i].addEventListener('click', (ev) => {
+    if (ev.target.tagName === 'IMG') {
+      ev.preventDefault();
+
+      const parentElement = ev.target.parentElement;
+
+      mainImage.src = parentElement.href;
```

Comment:
The `parentElement` is an `Element` object, and it does not have an `href` property. You should use `parentElement.getAttribute('href')` to access the `href` attribute of the parent element if it is an anchor tag.